docker/Dockerfile

```diff
@@ -0,0 +1,22 @@
+FROM openmicroscopy/devslave-c7:0.3.1
```

Comment:
Alongside #94, this can probably be bumped to `0.4.0` too. This can be done outside this PR.